### ISSUE TITLE
can't access my finetuned model

### ISSUE THREAD

**Issue by GP24CairoUni** [bclavie/RAGatouille#223]
from ragatouille import RAGTrainer

trainer = RAGTrainer(model_name="MyFineTunedColBERT",
                     pretrained_model_name="colbert-ir/colbertv2.0")

trainer.prepare_training_data(raw_data=positive_pairs,
                              data_out_path="./data2/",
                              all_documents=cleaned_documents,
                              mine_hard_negatives=True)

training_path = trainer.train(batch_size=16,
                              nbits=4,
                              maxsteps=500000,
                              use_ib_negatives=True,
                              dim=128,
                              learning_rate=5e-6,
                              doc_maxlen=256,
                              use_relu=False,
                              warmup_steps="auto")

![image](https://github.com/bclavie/RAGatouille/assets/157979917/2da0a5da-e977-4e2f-9292-6d67a910d15a)

I finished finetuning but I cant access and use the model again and no checkpoints on my local storage

**Comment by abdelkareemkobo** [comment]
may be try to export it into the huggingface hub and then download it again, but didn't find it in the path is extremely weird.
